common/config/data.tsx

```diff
@@ -5,6 +5,10 @@ import { GasPriceSetting } from 'types/network';
 import { makeExplorer } from 'utils/helpers';
 import NewTabLink from 'components/ui/NewTabLink';
 
+// tslint:disable-next-line:no-http-string
+export const IPFS_GATEWAY = 'http://127.0.0.1:8080';
```

Comment:
NOTE: Update before merge.

Reply:
@HenryNguyen5 we should consider having support for multiple endpoints in case one of them goes down.